src/main/java/net/minecraftforge/common/util/MutableHashedLinkedList.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+package net.minecraftforge.common.util;
```

Comment:
The `checkLicense` task reports this license header as invalid; please run the `updateLicenses`/`licenseFormat` task.